inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -3,11 +3,184 @@ title: "Vaccination survey"
 output: github_document
 ---
 
+
+# Introduction to this template
+This is a template which will guide you through the following steps:  
+  - merging household and individual level data
+  - calculating overall proportion of children <15 vaccinated against a specific pathogen
+  - calculating proportion of children <15 vaccinated against a specific pathogen by age group, location and gender
+  
+
 ```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
-library(dplyr)
-library(knitr)
+# hide all code chunks in the output, but show errors
+knitr::opts_chunk$set(echo = FALSE, error = TRUE, fig.width = 16, fig.height = 12)
+# set default NA to - in output, define figure width/height
+options(knitr.kable.NA = "-")
+
+
+library(knitr) # for creating output doc
+library(dplyr) # for cleaning/shaping data
+library(ggplot2) # for plotting diagrams
+library(rio) # for importing data
+library(tidyr) # for cleaning data
+library(epitrix) # for cleaning data
+
+# epi packages
+library(devtools) # for installing sitrep package
+library(incidence) # for epicurves
+library(ISOweek) # for creating epiweeks
+library(epitools) # for creating 2by2 tables
+library(survey)
+
+
+# devtools::install_github("R4EPI/sitrep")
+
+#set default text size to 16 for plots
+ggplot2::theme_set(theme_bw(base_size = 18))
+```
+
+
+```{r data import}
+# read in data
+# you can use rio to read in tabular data (Excel/csv) files
+# read in household data sheet
+vaccine_data_hh <- import("C:/Users/Patrick/Documents/GitHub/sitrep/test_vaccination_coverage_survey.xlsx", 
+                              which = "Level 0 Named")
+
+# read in individual level data sheet
+vaccine_data_indiv <- import("C:/Users/Patrick/Documents/GitHub/sitrep/test_vaccination_coverage_survey.xlsx", 
+                              which = "Level 1 Named")
+
+# read in data dictionary (for reference purposes)
+dictionary <- import("C:/Users/Patrick/Documents/GitHub/sitrep/test_vaccination_coverage_survey.xlsx",
+                      which = "Data Dictionary")
+
+# # read in population data
+# population <- import("your_data",
+#                      which = "sheetName")
+
+
 ```
 
 
-Just a placeholder at the moment.
+```{r data cleaning}
```

Comment:
consider following cholera template for cleaning steps - breaks it down by variable type